vhdbuilder/scripts/windows/automate_helpers.sh

```diff
@@ -47,25 +53,60 @@ create_pull_request() {
     #    ],
     #    "documentation_url": "https://docs.github.com/rest/pulls/pulls#create-a-pull-request"
     # }
-    result=$(curl -H "Authorization: token $github_access_token" \
+    if [[ $4 == "ReleaseNotes" ]]; then
+        result=$(curl -H "Authorization: token $github_access_token" \
         -H "Accept: application/vnd.github.v3+json" \
         "https://api.github.com/repos/Azure/AgentBaker/pulls?state=open&head=Azure:$3" |\
-        jq '.[] | select(.title == "chore: automated PR to bump windows image version to '$1'")')
-    if [[ -n $result ]]; then
-        echo "Pull request at head '$3' with title \"chore: automated PR to bump windows image version to '$1'\" existed already"
-        echo "Error: you cannot run image version bumping twice"
-        exit 1
+        jq '.[] | select(.title == "chore: automated PR to update windows release notes for version '$1'")')
+        if [[ -n $result ]]; then
+            echo "Pull request at head '$3' with title \"chore: automated PR to update windows release notes for version '$1'\" existed already"
+            number=$(echo $result | jq '.number')
+            echo "The existing pull request is at https://github.com/Azure/AgentBaker/pull/$number"
+            echo "Error: you cannot update release notes twice"
+            exit 1
+        else
+            response=$(curl -X POST \
+                -H "Authorization: token $github_access_token" \
+                -H "Content-Type: application/json" \
+                -d '{
+                    "title": "chore: automated PR to update windows release notes for version '$1'",
+                    "body": "This is an automated PR to update windows release notes for version '$1'",
+                    "head": "'$3'",
+                    "base": "master"
+                }' \
+                https://api.github.com/repos/Azure/AgentBaker/pulls)
+
+            number=$(echo $response | jq '.number')
+            echo "The pull request number is $number"
+            echo "The pull request link is https://github.com/Azure/AgentBaker/pull/$number"
+        fi
     else
-        curl -X POST \
-            -H "Authorization: token $github_access_token" \
-            -H "Content-Type: application/json" \
-            -d '{
-                "title": "chore: automated PR to bump windows image version to '$1'",
-                "body": "This is an automated PR to bump windows image version to '$1'",
-                "head": "'$3'",
-                "base": "master"
-            }' \
-            https://api.github.com/repos/Azure/AgentBaker/pulls
+        result=$(curl -H "Authorization: token $github_access_token" \
```

Comment:
duplicated code so it seems like that we can define a function and use it with different parameters.

Reply:
defined a function `curl_post_request` which is called in L101, `automate_helper.sh`. Test results (new queued pipelines: 1st run, 2nd run) are updated on wiki,